README.md

```diff
@@ -147,19 +88,17 @@ query = <<~GQL
   }
 GQL
 
+schema_cache_key = GraphqSchema.schema_cache_key
 
-payload = {
-  schema_cache_key: GraphqSchema.schema_cache_key,
-  query_cache_key: {
-    path_cache_key: ["post(id:1)", "cachedAuthor"],
-    selections_cache_key: { "cachedAuthor" => %w[id name] }
-  },
-}
+path_cache_key = "post(id:1)/cachedAuthor"
+selections_cache_key = "[#{%w[id name].join(".")}]"
```

Comment:
`selections_cache_key` can contain nested selection sets, e.g. `{ id, author: { name }`, how should it look like when joined with dots?

Reply:
It's gonna be `[id.author[name]]`.

See the original implementation from the gist: https://gist.github.com/palkan/faad9f6ff1db16fcdb1c071ec50e4190#file-03_fragment_caching-rb-L6-L11